### PR TITLE
PDI-19757] added a check to make sure that the file exists

### PR DESCRIPTION
**Pull request by MarcDiamantHitachi**

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 7m 25s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
:ok_hand: All tests passed!
 
Tests run: **15**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-8858/1/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java [hunk 1]
```diff
@@ -478,7 +480,9 @@ private void setPreviousSelection() {
         targetPath = targetPath.substring( 7 );
       }
       java.io.File filePath = new java.io.File( targetPath );
-      if ( filePath.isFile() ) {
+      if( !filePath.exists()) {
```
Comment (ddiroma) [inline plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java:483]: Please reformat (checkstyle)

File: plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java [hunk 2]
```diff
@@ -124,6 +124,8 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
```
Comment (ddiroma) [inline plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java:127]: These are unused imports, please remove

**Comment by buildguy** [comment]
### :x: Build failed in 7m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
:ok_hand: All tests passed!
 
Tests run: **15**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-8858/2/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Review by ddiroma — APPROVED** [review]

**Pull request merged**